scripts/globals/znm.lua

```diff
@@ -27,10 +27,16 @@ xi.znm = xi.znm or {}
 xi.znm.soultrapper = xi.znm.soultrapper or {}
 
 xi.znm.soultrapper.onItemCheck = function(target, user)
-    if not user:isFacing(target) then
+    -- Target checks.
+    if
+        target == nil or -- Players can use a macro to bypass the client side targeting restriction.
```

Comment:
This issue currently also exists for fellows lol. you can blast them with a nuke using macros